### DISCUSSION TITLE
Email "xxx@xxx.co.uk" does not comply with addr-spec of RFC 2822

### DISCUSSION THREAD

**Discussion by alexr8** [solspace/craft-freeform#668] [Questions & Answers]
**Describe the bug or issue you're experiencing**
I get a recurring error in the freeform error logs for the following:
```
Email "​xxxx@xxxx.co.uk" does not comply with addr-spec of RFC 2822. {"template":"vacancyApplicantNotification","file":"/srv/users/xxxx/apps/xxxx/releases/20230302170957/vendor/symfony/mime/Address.php"} []
```
I've looked through the other similar issues filed by others and re-checked the formatting the of the email template, and it seems to be calling things fields correctly (and follows similar patterns to other email templates i have that don't generate the same error). The only difference is this template is one that is sent to a submitter, linked via an email field in the form, whereas the other templates are sent to admins via the forms settings.

It is worth noting that the email it is complaining about doesn't have a valid inbox (its a noreply address) - however for all the other admin notification templates it doesn't generate an error about it, so instead i think it could be something related to this specific form / template rather than the email address itself?

Any chance you might be able to review my implementation and work out where the error is coming from?

**Steps to reproduce**

1. Create a user notification template, and link it via an email field in a form, using an email address that doesn't have an inbox
2. Submit the form on the front-end.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.4.8
- Freeform Version: 4.0.22
- Freeform Edition: Pro
- Fresh Install or Upgrade: upgrade (but this issue has existed since Craft 3)

**Comment by kjmartens** [comment]
Hi @alexr8,

I wonder if the issue is in how the `vacancyApplicantNotification` notification template is configured. Can you share that with us here (or create a private support ticket if you prefer)? 🙂

**Reply by binarymonkey84** [reply]
Hi @kjmartens 

Just jumping in here for @alexr8 as we're both working on the same project.

We're still seeing this problem.

The notification template is a database template. Relevant settings:

- "From Email" is set to an email address (not an environment variable). The email address is in a valid format (noreply@domain.co.uk) - actual domain name is redacted here for privacy
- "Reply-to Email" is set to a different email address, but it's still a valid one and not from an environment var either (e.g. recruitment@domain.co.uk)

Let me know if you need more information - we are obviously quite keen to resolve this issue as soon as possible.

**Reply by kjmartens** [reply]
Thanks @binarymonkey84,

The error mentioned says this:

`Email "​xxxx@xxxx.co.uk" does not comply with addr-spec of RFC 2822. {"template":"vacancyApplicantNotification","file":"/srv/users/xxxx/apps/xxxx/releases/20230302170957/vendor/symfony/mime/Address.php"} []`

This implies that it takes place in a file-based template with the name `vacancyApplicantNotification`. At any rate, it seems to think the email address is invalid. Are you 100% sure there are no spaces or special or "gremlin" characters set for its value (such as an invisible space, etc)? The easiest way to test for that would be to completely clear the entire value and type in the value again. 🙂

**Reply by binarymonkey84** [reply]
Yes, we have tried deleting the template and creating again, being careful to type the email out perfectly. I am 100% sure it's a valid format. 

The settings show it's a definitely a database template, not file. We don't have any file templates on this site. 

The template key value in the error message relates to the template slug / handle, it's not a path to any file.
